net/src/ipc/n3h.rs

```diff
@@ -0,0 +1,294 @@
+//! check that n3h is in the path, or download it
```

Comment:
Actually, I believe it would be much better to do some logging in here, not just in the error cases, but the success case, to let the user know what is going on here

Reply:
there is indeed some logging... like when [downloading](https://github.com/holochain/holochain-rust/pull/1137/files#diff-487ce26ffed9d1889698c2d916ad0162R223) or [executing commands](https://github.com/holochain/holochain-rust/pull/1137/files#diff-487ce26ffed9d1889698c2d916ad0162R135).

What else would you like to see?

Reply:
Ah cool yeah sorry my eyes were looking for println s